src/main/java/org/folio/innreach/config/JpaAuditingConfig.java

```diff
@@ -0,0 +1,18 @@
+package org.folio.innreach.config;
+
+import java.util.Optional;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.domain.AuditorAware;
+import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
+
+@Configuration
+@EnableJpaAuditing(auditorAwareRef = "auditorAware")
+public class JpaAuditingConfig {
+
+  @Bean
+  public AuditorAware<String> auditorAware() {
+    return () -> Optional.of("admin");
```

Comment:
Is it always going to be admin as a current user?

Reply:
Yes, this is a temp solution. Will be changed when okapi integration is ready

Reply:
ok, thanks